<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptEditorWindow</class>
 <widget class="QWidget" name="ScriptEditorWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>717</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Script Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: Helvetica, Arial, sans-serif;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="newButton">
       <property name="toolTip">
        <string>New Script (Ctrl+N)</string>
       </property>
       <property name="text">
        <string>New</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../resources/icons/new-script.svg</normaloff>
         <normalon>../resources/icons/new-script.svg</normalon>../resources/icons/new-script.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="loadButton">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Load Script (Ctrl+O)</string>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../resources/icons/load-script.svg</normaloff>../resources/icons/load-script.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveButton">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>32</width>
         <height>0</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Save Script (Ctrl+S)</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../resources/icons/save-script.svg</normaloff>../resources/icons/save-script.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRepeatDelay">
        <number>316</number>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toggleRunButton">
       <property name="toolTip">
        <string>Toggle Run Script (F5)</string>
       </property>
       <property name="text">
        <string>Run/Stop</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../resources/icons/start-script.svg</normaloff>../resources/icons/start-script.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="autoReloadCheckBox">
       <property name="styleSheet">
        <string notr="true">font: 13px &quot;Helvetica&quot;,&quot;Arial&quot;,&quot;sans-serif&quot;;</string>
       </property>
       <property name="text">
        <string>Automatically reload externally changed files</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>80</height>
      </size>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>ScriptEditorWindow</receiver>
   <slot>saveScriptClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toggleRunButton</sender>
   <signal>clicked()</signal>
   <receiver>ScriptEditorWindow</receiver>
   <slot>toggleRunScriptClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newButton</sender>
   <signal>clicked()</signal>
   <receiver>ScriptEditorWindow</receiver>
   <slot>newScriptClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadButton</sender>
   <signal>clicked()</signal>
   <receiver>ScriptEditorWindow</receiver>
   <slot>loadScriptClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>ScriptEditorWindow</receiver>
   <slot>tabSwitched(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>tabCloseRequested(int)</signal>
   <receiver>ScriptEditorWindow</receiver>
   <slot>tabCloseRequested(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
